src/main/java/org/folio/dao/transactions/TemporaryEncumbranceTransactionDAO.java

```diff
@@ -0,0 +1,53 @@
+package org.folio.dao.transactions;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.Transaction;
+import org.folio.rest.persist.Criteria.Criterion;
+import org.folio.rest.persist.DBClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.folio.rest.util.ResponseUtils.handleFailure;
+
+public class TemporaryEncumbranceTransactionDAO extends BaseTemporaryTransactionsDAO{
+
+  private static final String TEMPORARY_ENCUMBRANCE_TRANSACTIONS = "tmp_encumbered_transactions";
+  private static final String TEMPORARY_ENCUMBRANCE_TRANSACTIONS_QUERY = "SELECT jsonb FROM tmp_encumbered_transactions WHERE %s ";
```

Comment:
nitpick: considering referencing `TEMPORARY_ENCUMBRANCE_TRANSACTIONS` in `TEMPORARY_ENCUMBRANCE_TRANSACTIONS_QUERY` if "tmp_encumbered_transactions" should never diverge in both variables.

Reply:
Unfortunately there is no possibility, it is used in LedgerRolloverService after some changes

Reply:
sounds good.